applications/oci-transaction-lifecycle.md

```diff
@@ -111,10 +109,10 @@ If you've already started implementing your project or it is part of a larger re
 | Number | Deliverable | Specification |
```

Comment:
I noticed that the overview FTE spelled out "three" and then repeated it in parentheses as a numeral, but none of the other sections did this. Was there a reason that this was the only section that spelled out a numeral?